src/App.jsx

```diff
@@ -14,70 +17,60 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, updateGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood
+          ? (
+            <>
+              {`${selectedGood} is selected`}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+              <button
+                onClick={() => {
+                  updateGood('');
+                }}
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+              />
+            </>
+          )
+          : 'No goods selected'}
+      </h1>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+      <table className="table">
+        <tbody>
+          {goods.map(item => (
+            <tr data-cy="Good" className={`${selectedGood === item && 'has-background-success-light'}`}>
+              <td>
+                <button
+                  onClick={() => (
+                    selectedGood === item
+                      ? updateGood('')
+                      : updateGood(item)
+                  )}
```

Comment:
```suggestion
          {goods.map(good => {
            const isSelected = selectedGood = good;
            
            return (
            <tr data-cy="Good" className={cn({
              'has-background-success-light': isSelected,
            })}>
              <td>
                <button
                  onClick={() => setSelectedGood(isSelected ? '' : good)}
```
make variable isSelected and use it in all places.
suggest more meaningful naming for good
use classnames library for dynamic set classes